src/orders/components/OrderHistory/ExtendedTimelineEvent.tsx

```diff
@@ -122,28 +146,34 @@ const ExtendedTimelineEvent: React.FC<ExtendedTimelineEventProps> = ({
     id,
     date,
     type,
-    user,
     lines,
     amount,
     transactionReference,
-    shippingCostsIncluded,
-    relatedOrder
+    shippingCostsIncluded
   } = event;
   const classes = useStyles({});
   const intl = useIntl();
 
   const eventTypeInCamelCase = camelCase(type);
 
-  const employeeName = `${user.firstName} ${user.lastName}`;
+  const getTitleProps = () => {
```

Comment:
Use `if / if else` where possible and only consider using `switch case` when it makes things more readable in larger lists. Additionally, this function does not take any arguments and its outcome can be computed immediately inside render. Refactor to use:
```
const titleProps = type === OrderEventsEnum.ORDER_LINE_DISCOUNT_UPDATED ? { ... } : {};
```
If you _know_ this to get bigger very soon then leave a comment saying why you did it this way. Premature optimisation is the root of all devil so it's usually best to refactor this small piece of code when this situation presents itself.